apis/placement/v1alpha1/eviction_types.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster,categories={fleet,fleet-placement}
```

Comment:
does it need to be cluster scoped?

Reply:
Yes, since it's targeting a CRP and Placement i.e. CRB which are both cluster-scoped